linera-views/src/reentrant_collection_view.rs

```diff
@@ -569,35 +553,96 @@ where
     /// # use crate::linera_views::views::View;
     /// # let context = create_memory_context();
     ///   let mut view : ReentrantByteCollectionView<_, RegisterView<_,String>> = ReentrantByteCollectionView::load(context).await.unwrap();
+    ///   {
+    ///     let _subview = view.try_load_entry_mut(&[0,1]).await.unwrap();
+    ///   }
     ///   let short_keys = vec![vec![0, 1], vec![2, 3]];
     ///   let subviews = view.try_load_entries(short_keys).await.unwrap();
-    ///   let value1 = subviews[0].get();
-    ///   let value2 = subviews[1].get();
-    ///   assert_eq!(*value1, String::default());
-    ///   assert_eq!(*value2, String::default());
+    ///   assert!(subviews[1].is_none());
+    ///   let value0 = subviews[0].as_ref().unwrap().get();
+    ///   assert_eq!(*value0, String::default());
     /// # })
     /// ```
     pub async fn try_load_entries(
         &self,
         short_keys: Vec<Vec<u8>>,
-    ) -> Result<Vec<ReadGuardedView<W>>, ViewError> {
+    ) -> Result<Vec<Option<ReadGuardedView<W>>>, ViewError> {
+        let num_keys = short_keys.len();
+        let mut results = Vec::new();
+        for _ in 0..num_keys {
+            results.push(None);
+        }
         let mut updates = self.updates.lock().await;
-        let entries = Self::do_load_entries(
-            &self.context,
-            &mut updates,
-            self.delete_storage_first,
-            short_keys,
-        )
-        .await?;
-        entries
-            .into_iter()
-            .map(|(short_key, view)| {
-                Ok(ReadGuardedView(
-                    view.try_read_arc()
-                        .ok_or_else(|| ViewError::TryLockError(short_key))?,
-                ))
-            })
-            .collect()
+        let mut selected_indices = Vec::new();
+        let mut handles = Vec::new();
+        let mut test_indices = Vec::new();
+        for (i_key, short_key) in short_keys.iter().enumerate() {
+            match updates.entry(short_key.to_vec()) {
+                btree_map::Entry::Occupied(entry) => {
+                    let entry = entry.into_mut();
+                    if let Update::Set(_entry) = entry {
+                        selected_indices.push(i_key);
```

Comment:
Maybe `present_indices` or `existing_indices`?

Reply:
Ok for `present_indices`.